packages/autorest.go/package.json

```diff
@@ -63,8 +68,6 @@
     "@autorest/extension-base": "~3.5.2",
     "@azure-tools/codegen": "~2.9.2",
     "@autorest/codemodel": "~4.19.3",
-    "@azure-tools/codegen.go": "workspace:*",
```

Comment:
have you moved to pure `npm` worksapce? the `worksapce:` prefix is I thikn something only `pnpm` support(and rush if you configure to use pnpm workspace instead of the default system)

Reply:
if you moved to `pnpm` workspace then you should use `pnpm pack` instead of `npm pack` 
if you moved to `npm` workspace then it should be good to change to the correct version I think.

Reply:
Is the pure npm workspace what's described [here](https://docs.npmjs.com/cli/v10/using-npm/workspaces)?